lib/travis/api/v3/router.rb

```diff
@@ -1,3 +1,5 @@
+# require 'raven'
```

Comment:
In the cron model I changed line 16 in the `next_enqueuing` method from `Time.now` to `Time.nowt`, the spec (when run locally) fails because the Raven event that's raised doesn't have an `include?` method. 

I pushed to branch to staging and created a couple of crons, and noted the following errors being reported in Sentry:
![image](https://cloud.githubusercontent.com/assets/1754975/17171906/eed3fe24-53f2-11e6-9eb3-78c1c1e3d046.png)

So all looks good. I'll revert the typo then all should be good to merge.